docs/content/dagster-plus/managing-deployments/branch-deployments/change-tracking.mdx

```diff
@@ -0,0 +1,176 @@
+
+---
+
+## How it works
+
+Branch Deployments compare asset definitions in the branch deployment against the asset definitions in the main deployment. The UI will then mark changed assets accordingly. For example, if the pull request associated with the branch deployment adds a new asset, the UI will display a label indicating the addition.
+
+You can also apply filters to show only new and changed assets in the UI. This makes it easy to understand which assets will be impacted by the changes in the branch deployment.
+
+---
+
+## Supported change types
+
+Change tracking can detect the following changes to assets:
+
+- [New assets](#new-assets)
+- [Code versions](#code-versions)
+- [Upstream dependencies](#upstream-dependencies)
+- [Partitions definitions](#partitions-definitions)
+
+### New assets
+
+If an asset is new in the branch deployment, the asset will have a **New in branch** label in the asset graph:
+
+<Image
+src="/images/dagster-cloud/managing-deployments/change-tracking-new.png"
+width={764}
+height={308}
+/>
+
+### Code versions
+
+Change tracking can detect when an asset's `code_version` has changed, whether it's just been added, removed, or updated.
+
+Some Dagster integrations, like `dagster-dbt`, automatically compute code versions for you. For more information on code versions, including strategies for how to compute code versions automatically, refer to the [Code versioning guide](/guides/dagster/asset-versioning-and-caching).
+
+<TabGroup>
+<TabItem name="Asset definition">
+
+For example, if the main branch contains `customers` with code version `v1`
+
+```python file=/dagster_cloud/branch_deployments/change_tracking_code_version.py startafter=start_main_deployment endbefore=end_main_deployment dedent=4
+@asset(code_version="v1")
+def customers(): ...
+```
+
+And the pull request modifies `customers` and changes the code version to `v2`:
+
+```python file=/dagster_cloud/branch_deployments/change_tracking_code_version.py startafter=start_branch_deployment endbefore=end_branch_deployment dedent=4
+@asset(code_version="v2")
+def customers(): ...
+```
+
+---
+
+</TabItem>
+<TabItem name="Asset in the Dagster UI">
+
+Click the **Asset definition** tab to view how this asset is defined.
+
+In this example, the `customers` asset has a **Changed in branch** label indicating its `code_version` has been changed:
+
+<Image
+src="/images/dagster-cloud/managing-deployments/change-tracking-code-version.png"
+width={1042}
+height={444}
+/>
+
+---
+
+</TabItem>
+</TabGroup>
+
+### Upstream dependencies
+
+Change tracking can detect when an asset's upstream dependencies have changed, whether they've been added or removed.
+
+**Note**: If an asset is marked as having changed dependencies, it only means its dependencies have changed. It does **not** indicate that the upstream dependency has new data or that its definition has changed.
```

Comment:
The part I have been struggling with for this wording is that saying "its dependencies have changed" can be easily interpreted as "the upstream values have changed" 

Also the first sentence kind of reads like a self-referential definition. Maybe we can make it more specific by bringing asset keys into the mix? "If an asset is marked as having changed dependencies, it only means that the set of Asset Keys defining its upstream dependencies have changed"